Order.Management.PR/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Order.Management
+{
+    class Program
+    {
+        // Main entry
+        static void Main(string[] args)
+        {
+            var (customerName, address, dueDate) = CustomerInfoInput();
+
+            var orderedShapes = CustomerOrderInput();
+
+            InvoiceReport(customerName, address, dueDate, orderedShapes);
+
+            CuttingListReport(customerName, address, dueDate, orderedShapes);
+
+            PaintingReport(customerName, address, dueDate, orderedShapes);
+        }
+        
+        // Order Circle Input
+        public static Circle OrderCirclesInput()
+        {
+            Console.Write("\nPlease input the number of Red Circle: ");
+            int redCircle = Convert.ToInt32(userInput());
```

Comment:
- `int` should be replaced by `var`. The same improvement could apply to similar areas in PR. For more information, please read here: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables
- we should validate the user input before parsing it to an integer. I recommend extracting this logic and put it into a separate helper class (separate concern and unit testable)